rfcs/041-item-statuses/README.md

```diff
@@ -0,0 +1,311 @@
+Instead, we will create a new set of statuses, which will be mapped as follows:
+
+<table>
+  <tr>
+    <th>id</th>
+    <th>label</th>
+    <th>Sierra statuses</th>
+    <th>comment</th>
+  </tr>
+  <tr>
+    <td>available</td>
+    <td>Available</td>
+    <td><code>-</code> / Available</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>closed</td>
+    <td>Closed</td>
+    <td>
+      <code>h</code> / Closed <br/>
+    </td>
+    <td>
+      We distinguish between closed/restricted/unavailable in the access statuses.
+      I'm assuming it's a useful distinction here, but we could collapse this into "Unavailable" if not.
+    </td>
+  </tr>
+  <tr>
+    <td>restricted</td>
+    <td>Restricted</td>
+    <td>
+      <code>6</code> / Restricted <br/>
+    </td>
+    <td>
+      Ditto above.
+    </td>
+  </tr>
+  <tr>
+    <td>unavailable</td>
+    <td>Unavailable</td>
+    <td>
+      <code>r</code> / Unavailable <br/>
+      <code>m</code> / Missing <br/>
+      <code>x</code> / Withdrawn
+    </td>
+    <td>
+      I assume we wouldn't want to advertise if an item was missing.
+    </td>
+  </tr>
+  <tr>
+    <td>on-exhibition</td>
+    <td>On exhibition</td>
+    <td>
+      <code>e</code> / On exhibition
+    </td>
+    <td>
+    </td>
+  </tr>
+  <tr>
+    <td>on-display</td>
+    <td>On display</td>
+    <td>
+      <code>d</code> / On display
+    </td>
+    <td>
+    </td>
+  </tr>
+  <tr>
+    <td>in-cataloguing</td>
+    <td>In cataloguing</td>
+    <td>
+      <code>p</code> / In cataloguing
+    </td>
+    <td>
+      This feels analogous to the "on order" or "on exhibition" statuses.
+    </td>
+  </tr>
+  <tr>
+    <td>on-holdshelf</td>
+    <td>On holdshelf</td>
+    <td>
+      <code>!</code> / On holdshelf
+    </td>
+    <td>
+      Is the "shelf" part important?
+      Could this be "on-hold"?<br/><br/>
+      Do we need a way to signal "on hold for the logged-in user"?
+      We don't want to expose the user ID here, but knowing "it's on hold by someone" vs "it's on hold by you" might be useful.
+    </td>
+  </tr>
+  <tr>
+    <td>in-quarantine</td>
+    <td>In quarantine</td>
+    <td>
+      <code>t</code> / In quarantine
+    </td>
+    <td>
+      This is arguably only useful as long as COVID lasts, but it wouldn't be too tricky to add if we wanted it.
+      Normally this status is "in transit", but it seems to have been renamed temporarily.
+    </td>
+  </tr>
+  <tr>
+    <td>permission-required</td>
+    <td>Permission required</td>
+    <td><code>y</code> / Permission required</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>unknown</td>
+    <td>Unknown</td>
+    <td>(all others)</td>
+    <td>
+      This would be a catchall for all Sierra statuses we can't map to another status.
+    </td>
+  </tr>
+</table>
+
+As with the Sierra locations, we will match on the text description of the status, rather than the exact codes.
+This will make us a bit less coupled to the Sierra implementation.
```

Comment:
Honestly, I'm not sure this is a good idea. The text label is more likely to change. We don't do this for eg material types, why would we do it here?

I'd be careful of overgeneralising the fact that we had to do that for locations. I see that as papering over a data quality issue that was too big to take on, I'm not sure that's the case here.